app/forms/options_form.rb

```diff
@@ -0,0 +1,14 @@
+require 'reform/form/coercion'
+
+class OptionsForm < Reform::Form
+  feature Coercion
+  property :toc, type: Types::Form::Int,
```

Comment:
Hmmm. I remember having type related issues with using inclusion. But that may have been before I added the type cast. I'll try it again and see.